File: Volume 12 (1200 - 1299)/1279/1279.py
flag = 0
while True:
	try:
		n = int(input())
		if flag: print()
		if (n % 4 == 0 and n % 100 != 0) or n % 400 == 0:
			print("This is leap year.")
			if n % 15 == 0: print("This is huluculu festival year.")
			if (n % 55 == 0): print("This is bulukulu festival year.")
		elif n % 15 == 0: print("This is huluculu festival year.")
		else: print("This is an ordinary year")
		flag = 1
	except EOFError:
		break

File: Volume 12 (1200 - 1299)/1279/1279.rb
flag = false
while n = gets
	n = n.to_i
	puts '' if flag
	if (n % 4 == 0 and n % 100 != 0) or n % 400 == 0
		puts("This is leap year.")
		if n % 15 == 0
			puts("This is huluculu festival year.")
		end
		if (n % 55 == 0)
			puts("This is bulukulu festival year.")
		end
	elsif n % 15 == 0
		puts("This is huluculu festival year.")
	else
		puts("This is an ordinary year.")
	end
	flag = true
end